leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
 
-Zunächst stellt die fiskaltrust-Middleware sicher, dass alle Belege von einem Dritten neben dem Poscreator und dem Posoperator verarbeitet werden. Dies ist die organisatorische Implementierung der Sicherheit.
+Die Konformität wird durch die Kombination diverser Methoden und Komponenten erreicht.
 
-Als technische Implementierung der Sicherheit wird jeder Request und auch jede Response gehasht, wodurch die Datenintegrität der Anforderungen und Antworten sichergestellt wird. Um Unveränderlichkeit zu gewährleisten, wird ein weiterer Hash-Wert generiert, der sich auf den gesamten Anforderungs-Antwort-Zyklus bezieht, einschließlich der Identifizierung des Zyklus, des Betriebszeitpunkts, der vom Menschen lesbaren Belegnummer und der Hash-Werte von Anforderung, Antwort und vorherigem Empfang, der so genannte Belegs-Hash-Wert. Diese Verkettung des Empfangs-Hash-Werts bietet Unveränderlichkeit und bietet die Möglichkeit, alle Änderungen und auch das Löschen bei jeder vom Possystem bereitgestellten Aktion zu erkennen.
+Zunächst wird über die fiskaltrust.Middleware sichergestellt, dass alle Belege von einer dritten Partei (fiskaltrust) neben dem Kassenhersteller und dem Kassenhändler verarbeitet werden. Dies nennen wir die organisatorische Implementierung des Schutzes vor Manipulation.
 
+Zur technische Implementierung des Schutzes vor Manipulation wird jeder Request von dem Kassensystem und auch jeder Response von der fiskaltrust.Middleware gehasht, wodurch die Datenintegrität der Requests und Responses sichergestellt wird. Um nachweisen zu können, dass keine Manipulation stattgefunden hat, wird ein weiterer Hash-Wert erzeugt, der sich auf die gesamte Request-Response-Kette bezieht. Dieser wird aus folgenden Werten generiert:
+- der eindeutigen Identifikation der Kette
+- dem Erstellungszeitpunkt
+- der Belegnummer
+- der Hash-Werte des Request und Response 
+- sowohl als auch dem Hash-Wert des vorherigen Blocks. 
 
+Diese Verkettung bietet die Möglichkeit, Manipulationen sicher zu erkennen.
 
  ![receipt-chain](../product-service-description/compliance-as-a-service/media/receipt-chain.png)
 
- 
 
-Um das Risiko eines Angriffs auf die Kette ab dem letzten nicht verketteten Hashwert zu begrenzen, bietet fiskaltrust einen Mechanismus, der die aktuellen Daten in die fiskaltrust-Cloud spiegelt. Dieser Datenspiegel bietet die Möglichkeit, Angriffe zu erkennen, die an der Registrierkasse selbst nicht erkennbar wären.
+Um das Risiko einer Manipulation der Kette auf den letzten nicht verketteten Hash-Wert zu begrenzen, stellt fiskaltrust einen Mechanismus bereit, der die aktuellen Daten in die fiskaltrust.Cloud hochlädt und dadurch spiegelt. Diese Datenspiegelung ermöglicht es, Manipulationen zu erkennen, die an der Registrierkasse selbst nicht erkennbar wären.
```

Comment:
"Kette" --> "Queue" oder "ftQueue" (?)
"Registrierkasse" --> "Kassen- oder Aufzeichnungssystem"

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86